src/bootstrap.js

```diff
@@ -190,8 +190,16 @@ async function bootstrap(argv, config, cosmiconfigOptions = {}) {
   validateConfigAgainstSchema(configFile);
 
   // load both core and user plugins
-  // TODO: expand config file format to allow the user to supply an array of plugins here
-  const plugins = [];
+  const userPlugins = configFile.config.plugins || [];
+  let plugins = [];
+  for (let plugin of userPlugins) {
+    if (plugin.startsWith("npm:")) {
```

Comment:
Not sure about `npm:` on reflection.
What if this came from https://jsr.io/ ?
What would be a better word here? Module? Package?

Naming things is hard

Reply:
I think package is a better word